doc/user/upgrade/upgrade_guide.md

```diff
@@ -22,6 +22,81 @@ $ kubectl edit deployment/etcd-operator
 ### Incompatible upgrade
 In the case of an incompatible upgrade, the process requires restoring a new cluster from backup. See the [incompatible upgrade guide](incompatible_upgrade.md) for more information.
 
+
+## v0.5.x -> v0.6.0
+
+### Breaking Change
+
+The v0.6.0 release removes [operator level s3 backup](https://github.com/coreos/etcd-operator/blob/v0.5.1/doc/user/backup_config.md#operator-level-configuration).
+
+If you are using operator level s3 backup, You must migrate your cluster to use [cluster level backup](../backup_config.md#s3-on-aws) for v0.6.0.
+
+Steps to upgrade:
+
+Assuming your cluster uses [operator level s3 backup](https://github.com/coreos/etcd-operator/blob/v0.5.1/doc/user/backup_config.md#operator-level-configuration).
+
+- Create a backup of you current data using the [backup service guide](../backup_service.md#http-api-v1).
+
+- Create a new AWS secret `newAws` for [cluster level backup](../backup_config.md#s3-on-aws):
+
+  `$ kubectl -n <namespace> create secret generic newAws --from-file=$AWS_DIR/credentials --from-file=$AWS_DIR/config`
+
+- Change the backup field in your existing Cluster backup spec to enable [cluster level backup](../backup_config.md#s3-on-aws):
+
+  From
+
+  ```yaml
+  backup:
+    backupIntervalInSecond: 30
+    maxBackups: 5
+    storageType: "S3"
+  ```
+
+  to
+
+  ```yaml
+  backup:
+    backupIntervalInSecond: 30
+    maxBackups: 5
+    storageType: "S3"
+    s3:
+      s3Bucket: <your-s3-bucket>
+      awsSecret: newAws
+  ```
+
+- Update existing cluster:
+  `$ kubectl -n <namespace> apply -f <your-cluster-deployment>.yaml`
+
+- Update etcd-operator spec to use `etcd-operator:v0.6.0` and remove dependency on operator level S3 backup flags:
+
+  From
+
+  ```yaml
+  spec:
+    containers:
+    - name: etcd-operator
+      image: quay.io/coreos/etcd-operator:v0.5.2
+      command: 
+        - /usr/local/bin/etcd-operator
+        - --backup-aws-secret=aws
+        - --backup-aws-config=aws
+        - --backup-s3-bucket=<your-s3-bucket>
+  ```
+
+  to 
+
+  ```yaml
+  spec:
+    containers:
+    - name: etcd-operator
+      image: quay.io/coreos/etcd-operator:v0.6.0
+      command: 
+        - /usr/local/bin/etcd-operator
+  ```
+
+- Update existing etcd-operator deployment:
```

Comment:
This is duplicate to above point. Remove this?

Reply:
the above ask you to update the config. this line ask you to apply the config change